test/powershell/Modules/Microsoft.PowerShell.Management/Add-Content.Tests.ps1

```diff
@@ -59,3 +59,37 @@ Describe "Add-Content cmdlet tests" -Tags "CI" {
     }
   }
 }
+
+Describe "Add-Content feature tests" -Tag Feature {
```

Comment:
We always have problems with async tests. Do we really need the test? I'd only add an comment in C# code.

Reply:
I kind of agree on async tests being inherently fragile.  Perhaps for this one a comment in the code is sufficient.

Reply:
Make it sync:
1) open a file with `FileStream(FileAccess.Read, FileShare.ReadWrite)` and keep it open.
2) run new `Add-Content` (with `FileAccess.Write, FileShare.ReadWrite`)
3) close `FileStream` in 1). The test should not throw.

The old `Add-Content` (with `FileShare.Write`) in 2) would throw.

Reply:
Good suggestion, let me see if that repros the original issue.

Reply:
Please see if these changes are possible. It would harden the test.

Reply:
@anmenaga that worked perfectly and so simple :)